### PR TITLE
Move over to login-only version of webapp

### PR DESCRIPTION
**Pull request by BruceJi7**
Login-only:
Remove all of the non-login code
Add functions to communicate with extension
Add 'you have logged in' page
Add sign out page

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -9,16 +9,20 @@ import Sidebar from "../Sidebar";
 import DeerIcon from "@/assets/icons/metacon-deer-blue.svg";
 
 import styles from "./Header.module.scss";
+import { useRouter } from "next/router";
```
Comment (BruceJi7) [inline src/components/Header/index.tsx:12]: import order

File: src/pages/Login/ImportBackup/index.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import useStore from "@/store/store";
 import { getIdentity } from "@/helpers/indentity";
 
 import styles from "./ImportBackup.module.scss";
+import { useRouter } from "next/router";
```
Comment (BruceJi7) [inline src/pages/Login/ImportBackup/index.tsx:9]: import order

**Review by BruceJi7 — COMMENTED** [review]
a few import orders wrong

**Review by JM066 — APPROVED** [review]
Eveything looks good!

**Pull request merged**